src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -242,6 +242,7 @@
 
 (defn- internal-discover-view
   [params]
+  (rf/dispatch [:fetch-contract-communities])
```

Comment:
This dispatch is not related to the **Test Networks Enabled** option.

Just added back the curated communities fetch event dispatch that got removed in a [previous PR](https://github.com/status-im/status-mobile/pull/17094/files#diff-427a453633106ea987e59e011963a453be85a66a9cb88f34359346c35287e382L228).